source/content/secure-runtime-access.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Secure Runtime Access
+description: Limit access to your site's database to help defend against traffic-based attacks and unauthorized access.
+tags: [security]
+categories: []
+contributors: [edwardangert]
+searchboost: 150
+---
+
+Pantheon’s database services use strong random passwords and TLS to encrypt communications by default. Customers seeking additional defense in depth can enable Secure Runtime Access (SRA).
+
+When SRA is enabled, the the site's Pantheon MySQL databases will no longer listen for incoming connections over public internet ports, and developers will only be able to connect to those databases [via SSH tunnel](/ssh-tunnels/).
+
+In addition to defense in depth, this feature can be used to enforce roles and permissions by preventing developers from accessing a live database. It also guarantees that users who are removed from a site team or organization can no longer use a saved set of credentials.
```

Comment:
Suggest a stronger connection for [Developers](https://pantheon.io/docs/change-management#organizations-roles-and-permissions) as a specific role in an organization. At first read, I mistook it as a general assertion vs role-specific